release-habushu.sh

```diff
@@ -19,6 +19,13 @@ if [[ "$?" -ne 0 ]] ; then
   echo 'Process failed! Unable to check out release branch!'; exit 1
 fi
 
+echo "///////////  Bootstrap Habushu plugin ///////////"
+mvn clean install -Pbootstrap
```

Comment:
I: Allows `scm:check-local-modification` and `versions:set` to run as the SNAPSHOT version isn't always released.